x/ethbridge/types/msgs.go

```diff
@@ -270,21 +254,14 @@ func (msg MsgUpdateCethReceiverAccount) GetSignBytes() []byte {
 
 // GetSigners defines whose signature is required
 func (msg MsgUpdateCethReceiverAccount) GetSigners() []sdk.AccAddress {
-	return []sdk.AccAddress{msg.CosmosSender}
-}
-
-// MsgRescueCeth transfer the ceth from ethbridge module to an account
-type MsgRescueCeth struct {
-	CosmosSender   sdk.AccAddress `json:"cosmos_sender" yaml:"cosmos_sender"`
-	CosmosReceiver sdk.AccAddress `json:"cosmos_receiver" yaml:"cosmos_receiver"`
-	CethAmount     sdk.Int        `json:"ceth_amount" yaml:"ceth_amount"`
+	return []sdk.AccAddress{sdk.AccAddress(msg.CosmosSender)}
 }
 
 // NewMsgRescueCeth is a constructor function for NewMsgRescueCeth
 func NewMsgRescueCeth(cosmosSender sdk.AccAddress, cosmosReceiver sdk.AccAddress, cethAmount sdk.Int) MsgRescueCeth {
 	return MsgRescueCeth{
-		CosmosSender:   cosmosSender,
-		CosmosReceiver: cosmosReceiver,
+		CosmosSender:   cosmosSender.String(),
+		CosmosReceiver: cosmosReceiver.String(),
```

Comment:
Please change all strings that are sifchain addresses to be of type sdk.address

Reply:
why? strings are used because of issues with support in grpc and grpc-gateway.